jena-arq/src/main/java/org/apache/jena/sparql/util/Context.java

```diff
@@ -439,13 +434,36 @@ public static AtomicBoolean getCancelSignal(Context context) {
         }
     }
 
+    /** Atomic compute. */
```

Comment:
Haven't I seen these Context changes in another of your PRs?

Reply:
Its also part of the ExecTracker PR #3184 - but it also fits here where it is easier to review due to the size of the PR :)